### ISSUE TITLE
undefined method `bad_request?' during POST requests

### ISSUE THREAD

**Issue by adamrdrew** [adamrdrew/vueck#1]
Not 100% sure this is a VueCK problem yet, but documenting it here just in case. Calls to POST routes are bombing out with backtrace like this:

```ruby
Rack::Lint::LintError: Status must be >=100 seen as integer
        /mnt/c/Users/adamr/Development/personal/lilbloggy/vendor/bundle/ruby/2.3.0/gems/rack-2.0.4/lib/rack/lint.rb:20:in `assert'
        /mnt/c/Users/adamr/Development/personal/lilbloggy/vendor/bundle/ruby/2.3.0/gems/rack-2.0.4/lib/rack/lint.rb:620:in `check_status'
        /mnt/c/Users/adamr/Development/personal/lilbloggy/vendor/bundle/ruby/2.3.0/gems/rack-2.0.4/lib/rack/lint.rb:51:in `_call'
        /mnt/c/Users/adamr/Development/personal/lilbloggy/vendor/bundle/ruby/2.3.0/gems/rack-2.0.4/lib/rack/lint.rb:37:in `call'
        /mnt/c/Users/adamr/Development/personal/lilbloggy/vendor/bundle/ruby/2.3.0/gems/rack-2.0.4/lib/rack/show_exceptions.rb:23:in `call'
        /mnt/c/Users/adamr/Development/personal/lilbloggy/vendor/bundle/ruby/2.3.0/gems/rack-2.0.4/lib/rack/common_logger.rb:33:in `call'
        /mnt/c/Users/adamr/Development/personal/lilbloggy/vendor/bundle/ruby/2.3.0/gems/sinatra-2.0.0/lib/sinatra/base.rb:231:in `call'
        /mnt/c/Users/adamr/Development/personal/lilbloggy/vendor/bundle/ruby/2.3.0/gems/rack-2.0.4/lib/rack/chunked.rb:54:in `call'
        /mnt/c/Users/adamr/Development/personal/lilbloggy/vendor/bundle/ruby/2.3.0/gems/rack-2.0.4/lib/rack/content_length.rb:15:in `call'
        /mnt/c/Users/adamr/Development/personal/lilbloggy/vendor/bundle/ruby/2.3.0/gems/rack-2.0.4/lib/rack/handler/webrick.rb:86:in `service'
        /usr/lib/ruby/2.3.0/webrick/httpserver.rb:140:in `service'
        /usr/lib/ruby/2.3.0/webrick/httpserver.rb:96:in `run'
        /usr/lib/ruby/2.3.0/webrick/server.rb:296:in `block in start_thread'
[2018-02-05 21:51:11] ERROR NoMethodError: undefined method `bad_request?' for #<Rack::ShowExceptions:0x000000042c3680>
        (erb):145:in `pretty'
        /usr/lib/ruby/2.3.0/erb.rb:864:in `eval'
        /usr/lib/ruby/2.3.0/erb.rb:864:in `result'
        /mnt/c/Users/adamr/Development/personal/lilbloggy/vendor/bundle/ruby/2.3.0/gems/rack-2.0.4/lib/rack/show_exceptions.rb:96:in `pretty'
        /mnt/c/Users/adamr/Development/personal/lilbloggy/vendor/bundle/ruby/2.3.0/gems/rack-2.0.4/lib/rack/show_exceptions.rb:32:in `rescue in call'
        /mnt/c/Users/adamr/Development/personal/lilbloggy/vendor/bundle/ruby/2.3.0/gems/rack-2.0.4/lib/rack/show_exceptions.rb:23:in `call'
        /mnt/c/Users/adamr/Development/personal/lilbloggy/vendor/bundle/ruby/2.3.0/gems/rack-2.0.4/lib/rack/common_logger.rb:33:in `call'
        /mnt/c/Users/adamr/Development/personal/lilbloggy/vendor/bundle/ruby/2.3.0/gems/sinatra-2.0.0/lib/sinatra/base.rb:231:in `call'
        /mnt/c/Users/adamr/Development/personal/lilbloggy/vendor/bundle/ruby/2.3.0/gems/rack-2.0.4/lib/rack/chunked.rb:54:in `call'
        /mnt/c/Users/adamr/Development/personal/lilbloggy/vendor/bundle/ruby/2.3.0/gems/rack-2.0.4/lib/rack/content_length.rb:15:in `call'
        /mnt/c/Users/adamr/Development/personal/lilbloggy/vendor/bundle/ruby/2.3.0/gems/rack-2.0.4/lib/rack/handler/webrick.rb:86:in `service'
        /usr/lib/ruby/2.3.0/webrick/httpserver.rb:140:in `service'
        /usr/lib/ruby/2.3.0/webrick/httpserver.rb:96:in `run'
        /usr/lib/ruby/2.3.0/webrick/server.rb:296:in `block in start_thread'
```

GET requests work fine, which is why I didn't notice this until now.

**Comment by adamrdrew** [comment]
Yup, confirmed the problem is VueCK. Disabled it and posts are working fine. Curious!

**Comment by adamrdrew** [comment]
Fixed in a6b90ffa25db
